src/sagemaker_inference/encoder.py

```diff
@@ -87,6 +87,9 @@ def _array_to_csv(array_like):
 }
 
 
+SUPPORTED_CONTENT_TYPES = set(_encoder_map.keys())
```

Comment:
Couple of things I'm unsure about: 
* Should this constant be defined in `encoders`, `default_inference_handler`, or `transformer`?
* Should this constant be hard-coded, or is it better to use the `_encoder_map` keys?

Reply:
* I would put it in `default_inference_handler` because the "supported" nature of it is specific to the default handler's implementation of `output_fn`
* I think the keys are fine since it's intended to catch all of them.

Reply:
If we place `SUPPORTED_CONTENT_TYPES` in `default_inference_handler`, then we can't access the keys of `_encoder_map` because `_encoder_map` is internal to the `encoder` module.

Reply:
oh right. then I'd say leave it in this file, and keep the `encoder_map.keys()`